components/processing/README.md

```diff
@@ -36,5 +36,5 @@ Running invoke.sh will do the following -
 
 ### Deployments
 There are Python-based deployments, such as in [deployments/cloud_function/main.py](deployments/cloud_function/main.py) which leverage one or more libraries. There is an associates requirements.in and requirements.txt for these libraries (using -r) capturing the full non-library dependencies. It is assumed that all of the relevant libraries will be staged in the deployment.
-The instructions for Cloud Function deployment is available in the [README.md](deployments/cloud_function/README.md) and for Cloud Run is available in the [README.md](deployments/cloud_run/README.md).
+The instructions for Cloud Function deployment is available in the [README.md](deployments/cloud_function/README.md) and for Cloud Run is available in the [README.md](terraform/README.md).
```

Comment:
Do we still have this support deployment to cloud_function? should we get rid completely of the `deployments` folder?

Reply:
My vote is to remove the deployment of the customer document processor to cloud_function. Let's just use Cloud Run. If a customer or partner wants to deploy it on cloud function or GKe.. they can do it themselves.

Reply:
`deployments` folder removed